web/src/components/pages/contribution/contribution.css

```diff
@@ -1,9 +1,5 @@
 @import url('../../media.css');
 
-.contribution-wrapper {
-    overflow: hidden;
-}
-
```

Comment:
I think we should replace this with 

```overflow-x: hidden```

I was able to scroll horizontally on the speak/listen pages after recording/validating some clips

https://user-images.githubusercontent.com/113194143/235129495-d984b333-34ee-4002-8d4a-fac7b337bb4d.mov

Reply:
Restored, safer that way indeed :)

Reply:
I'm unsure if you can reproduce this issue but with `overflow: hidden` I cannot scroll fully when the screen is resized (check the attached video). This is fixed when I use `overflow-x: hidden`. Do you notice the same behaviour?

https://user-images.githubusercontent.com/113194143/235159398-0ed85299-45bd-4274-9d4d-61d231b0d23d.mov